app/controllers/games_controller.rb

```diff
@@ -39,10 +43,9 @@ def set_game
   end
 
   def set_games
-    @game_sections = SpeedrunDotComGame.order(name: :asc).slice_when do |a, b|
-      a.name[0].downcase != b.name[0].downcase
-    end.map do |srdc_games|
-      [srdc_games.first.name[0].downcase, srdc_games]
-    end.to_h
+    @games = Hash.new([])
+    SpeedrunDotComGame.order('ASCII(name) ASC').each do |game|
```

Comment:
@jhobz This `ORDER BY ASCII(name)` should fix the issue with games like .hack appearing in the `h` section. I tried to group all the symbols together into one section but it was too much complexity for the payoff, so there will be one section per symbol.